### PR TITLE
Fixes & additional updates

### PR DESCRIPTION
**Pull request by stephanecopin**
### Goals :soccer:

This PR:
- Add an optional `insets` parameter to `addAndFitSubview()`, which should be self-explanatory
- Make `removeArrangedSubviews()`'s `removeFromHierachy` parameter default to `true`
- Fix an internal state corruption issue in `OrderedSet`, and make it conform to `SetAlgebra`
- Add `tapped` helper to link any `Action` to any `UIControl`
- Add `AnyIdentifiable` & `AnyAction` for type-erased `Identifiable` & `ReactiveActionProtocol` respectively
- Add `OverridingAction`, a new `Action` that if executed when already executing, will cancel the current one and start anew

### Backwards-compatibility:

All changes are additive and doesn't break backward compatibility.

### Testing Details :mag:

Unit tests were added for `OrderedSet`.

### CODE REVIEW

File: FueledUtils/ReactiveSwiftUIKit/ControlProtocol+Tapped.swift [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+private var tapActionKey: UInt8 = 0
+
+extension Reactive where Base: ControlProtocol {
+	///
+	/// The action to be triggered when the button is tapped.
+	/// This mirrors the `pressed` property native in `ReactiveCocoa`, but uses a
+	/// protocol to represents the button rather than hardcode it to classes,
+	/// allowing for any `UIControl` to use this method.
+	///
+	var tapped: TapAction<Base>? {
```
Comment (stephanecopin) [inline FueledUtils/ReactiveSwiftUIKit/ControlProtocol+Tapped.swift:28]: I'd welcome a new name here, as `pressed` is already used, and they semantically are pretty much the same

**Review by metatheoretic — APPROVED** [review]
💯

File: FueledUtils/ReactiveSwift/AnyAction.swift [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	private let applyClosure: (Input) -> SignalProducer<Any, Error>
+	private let deinitToken: Lifetime.Token
+
+	init<Action: ReactiveActionProtocol>(_ action: Action) {
+		self.isEnabled = action.isEnabled
+		self.isExecuting = action.isExecuting
+		self.applyClosure = { action.apply($0 as! Action.Input).map { $0 }.mapError { $0 } }
+		self.events = action.events.map { $0.map { $0 }.mapError { $0 } }
+		self.values = action.values.map { $0 }
+		self.errors = action.errors.map { $0 }
```
Comment (harishsaini) [inline FueledUtils/ReactiveSwift/AnyAction.swift:32]: can we write 
```
self.values = action.values
self.errors = action.errors
```

is .map { $0 } required ?

Comment (stephanecopin) [reply FueledUtils/ReactiveSwift/AnyAction.swift:32]: @harishsaini it is, to convert to `Any` or `Error` through type inference. I could have added `as Any` in the `map`s and `as Error` in the `mapError`, but I figured it wasn't necessary as it's a common-ish pattern in Swift

**Comment by tinman-fueled** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please include a CHANGELOG entry to credit yourself! <br />
You can find it at <a href="https://github.com/Fueled/ios-utilities/blob/develop/CHANGELOG.md">CHANGELOG.md</a>.</td>
    </tr>
  </tbody>
</table>

Here's an example of your CHANGELOG entry:
```markdown
- Fixes & additional updates  
  [stephanecopin](https://github.com/stephanecopin)
  [#issue_number](https://github.com/CocoaPods/CocoaPods/issues/issue_number)
```
*note*: There are two invisible spaces after the entry's text.


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Pull request merged**